### DISCUSSION TITLE
Is there a way to connect the Greek Tax System called myData ?

### DISCUSSION THREAD

**Discussion by AlexPapaxen** [YetiForceCompany/YetiForceCRM#17093] [How to]
Hi, I would like to connect yetiforce with myData, the greek tax system. Is there any way to do this ? Maybe a future add-on would be useful.
Thank you a lot in advance !

**Comment by mariuszkrzaczkowski** [comment]
describe more precisely:
- what scope of integration
- how exactly would it work?
- what data is to be exchanged, directions
- some materials/information on how to integrate

**Reply by AlexPapaxen** [reply]
myData is the formal greek tax system, that was built recently by the greek goverment. It has a lot of features, the most important of them is to upload to the system of myData invoices, receipts, payments etc. The data then will be evaluated by the tax system. In general this is how it works. I do not know if the goverment provides maybe a free API to integrate with CRMs.
